src/utils/getTestFrameworkName.ts

```diff
@@ -2,12 +2,12 @@
 import type { PackageJSON } from "@nodesecure/npm-types";
 
 // CONSTANTS
-const kNodeTestRunnerRegex = /(?:node\s+--test|tsx)/;
+const kNodeTestRunnerRegex = /[node|tsx]\s+.*--test/;
 
 export function getTestFrameworkName(packageJson: PackageJSON): string {
   const { devDependencies: deps = {}, scripts = {} } = packageJson;
 
-  if (kNodeTestRunnerRegex.test(scripts.test)) {
+  if (kNodeTestRunnerRegex.test(scripts.test) || isUsingNodeTestRunner(scripts, "test")) {
```

Comment:
```suggestion
  if (isUsingNodeTestRunner(scripts, "test")) {
```
Since there is
```ts
if (scriptCommand && kNodeTestRunnerRegex.test(scriptCommand)) {
  return true;
}
```